src/main/java/onlinebookstore/controller/BookController.java

```diff
@@ -31,11 +34,13 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
+    @ResponseStatus(HttpStatus.CREATED)
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookRequestDto) {
```

Comment:
Optional:
you may also validate such things as @PathVariable, @RequestParams etc. It looks like:

`@PathVariable @Positive Long id`
In order to be able to perform such validation you need to annotate your controller class with @Validated annotation